src/dev/src/app/components/tree-view/tree-view-client-side-demo.ts

```diff
@@ -6,15 +6,16 @@ import {
   PagedTreeNodeDataRetriever,
```

Comment:
Your demo page for treeview starts being really long. Can you please create multiple sub-pages dedicated to each aspects of the treeview  ? 
Like one page for client side, one page for server side, one page for non-root and a dashboad page that explain globally what is the treeview and how to use it, it will also contain all component API (all @Input() you have in your component needs to be explained and documented). You can check what i did for the datagrid.
We'll need to review and write more precise/useful doc... I've planned to do it for all other components at some point but if you can do it for the treeview it would be more than appreciated.

Reply:
You're right, I separated it out now, theres two pages for them now each containing 2 examples. When I add more examples in the future i'll create more pages too